app/main/utils.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime
+
+from config import Config
+
+
+def calculate_days_to_deadline(current_date=datetime.date(datetime.now())):
+    """Calculate the number of days remaining until a specified submission deadline.
+    The due_date is a str representation of submission deadline in format dd/mm/yyyy.
+    It is set in main/config/envs/default.py
+
+    :param current_date: datetime object representing today's date in format yyyy-mm-dd HH:MM:SS
```

Comment:
is this correct, or should it now just be dd/mm/yyyy ?

Reply:
and this one